src/hal/src/command/raw.rs

```diff
@@ -209,7 +209,18 @@ pub trait RawCommandBuffer<B: Backend>: Any + Send + Sync {
     /// will operate on.
     ///
     /// Each buffer passed corresponds to the vertex input binding with the same index,
-    /// starting from an offset index `first_binding`.
+    /// starting from an offset index `first_binding`. For example an iterator with
+    /// two items and `first_binding` of 1 would fill vertex buffer binding numbers
+    /// 1 and 2.
+    ///
+    /// This binding number is local to the currently bound pipeline and is separate
```

Comment:
I don't think this is correct to say. It reads as if the binding gets reset upon changing the pipeline. The binding is just separate from descriptor sets, and pipeline knows how to fetch the data from those bindings.

Reply:
Yeah that wording was a bit bad. Fixed.